react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -15,22 +18,34 @@ export default {
       }
     }
     */
-    if (!('pinentry' in payload)) {
-      console.error('Passphrase payload has no pinentry object.')
-    } else if (!('features' in payload.pinentry)) {
-      console.error('payload.pinentry.features does not exist')
-    } else if (!('prompt' in payload.pinentry)) {
-      console.error('payload.pinentry.prompt does not exist')
-    }
+    const props = payload.pinentry
+    var pinentryWindow = new BrowserWindow({
+      width: 500, height: 300,
+      resizable: false,
+      fullscreen: false
+    })
+    pinentryWindow.hide()
+    pinentryWindow.loadUrl(`file://${__dirname}/pinentry.wrapper.html`)
 
-    var props = payload.pinentry
-    // Here's where we should spawn a BrowserWindow with the Pinentry component,
-    // passing in props.  For now, it's stubbed out.
-    const reply = {
-      'passphrase': 'fooBARbaz',
-      'storeSecret': false
-    }
-    response.result(reply)
-    console.log('Sent passphrase back')
+    ipc.on('pinentryNeedProps', function (event, arg) {
+      event.sender.send('pinentryGotProps', props)
```

Comment:
@MarcoPolo Looks like pinentryWindow is type BrowserWindow and event.sender is type EventEmitter, will need to try something else.